group.py

```diff
@@ -1,5 +1,32 @@
-"""An example of how to represent a group of acquaintances in Python."""
-
-# Your code to go here...
-
-my_group =
+friends = {
+    'Jill' : {
+        "age": 26, 
+        "job": "Biologist", 
+        "relation" : {
+            "friend":"Zalika",
+            "partner":"John"
+            },
+        },
+    'Zalika' : {
+        "age": 28, 
+        "job": "Artist", 
+        "relation" : {
+            "friend":"Jill"
+            },
```

Comment:
I may be overthinking it as a "high IQ move", but you could say that Zalika is Nash's tenant?

Apart from that a very nice simple solution :)